server/webapp/WEB-INF/rails.new/spec/new_javascripts/views/pipeline_configs/pipeline_config_widget_spec.js

```diff
@@ -63,6 +63,14 @@ define([
       expect(pipeline.enablePipelineLocking()).toBe(true);
     });
 
+    it("should render the pipeline scheduling type in the pipeline settings view", function() {
+      expect($root.html()).toContain('Automatically triggered');
```

Comment:
you can use `jasmine-jquery.js` then it becomes `.toContainHtml("Automatically triggered")`.